R/db-odbc-oracle.R

```diff
@@ -0,0 +1,235 @@
+#' @export
+sql_select.Oracle<- function(con, select, from, where = NULL,
```

Comment:
It feels like we're getting a lot of duplicated code in this selection methods. Maybe it's time to break out smaller methods `sql_select_select`, `sql_select_where`, `sql_select_group_by` etc?